CIP-dapp-bridge-rework/README.md

```diff
@@ -0,0 +1,1169 @@
+
+#### `JSON`
+
+[JavaScript Object Notation](https://json-schema.org/draft/2020-12/json-schema-core) is a lightweight data-interchange format that is easy for humans to read and write, and easy for machines to parse and generate. It is based on a subset of the JavaScript Programming Language and is commonly used for transmitting data in web applications and for storing configuration settings.<br><br>
+
+#### `URI`
+
+A URI image (e.g. data URI base64 or other) for img src for the wallet which can be used inside the dApp for the purpose of asking the user which wallet they would like to connect with.<br><br>
+
+#### `BigIntString`
+
+A BigInt number, stored as a string value. For example, the amount of something. It may be negative (e.g., for token burning).<br><br>
+
+#### `HexString`
+
+Byte data, converted to hex and stored as a string value. For example, the hash of something. Note, this is not the same as cbor<T>, although there is no difference in representation between them, there is a difference at the level of context and use cases!<br><br>
+
+#### `base58`
+
+A base58-encoded hexadecimal value. Typically used for Byron addresses.<br><br>
+
+#### `AddressString`
+
+A string representing an address in either bech32 or base58 format. Not CBOR!<br><br>
+
+#### `LocalString`
+
+_**Array**_ of objects describing a set of localizations of the same text, intended for the wallet user.
+
+```ts
+type LocalString = {
+  local: string,
+  body: string
+}[];
+```
+
+* **`local`**
+  
+  A string containing the locale code in the format [ISO-639.1](https://en.wikipedia.org/wiki/ISO_639-1).
+  
+  > **Note:** The DApp is not obliged to provide messages in languages it does not support.
+  
+* **`body`**
+
+  A string containing the localized text of the message. It is important that all `body` within one `LocalString` contain the same information. <br><br>
+
+The logic for working with `LocalString` is as follows:
+
+1. If a `LocalString` consisting of only one element is passed to the wallet, it is displayed "as is".
+2. If a `LocalString` consisting of several elements is passed to the wallet, the wallet should choose the one whose `local` is most suitable for the user.
+3. If the `LocalString` does not contain a localization suitable for the user, the wallet may choose any other localization it considers "default". In most cases, this will likely be "en". <br><br>
+
+
+#### `AccountEra`
+
+A string indicating whether the account belongs to the `Byron` or `Shelley` eras of Cardano.
+
+```ts
+type AccountEra = "Byron" | "Shelley";
+```
+
+#### `AccountId`
+
+An arbitrary string that serves as the identifier of the wallet account connected to the DApp, formed by the wallet according to its own rules. The main point is that the wallet understands what is required of it when it receives this identifier in a request.
```

Comment:
There is no notion of account on Cardano. If you mean account id in the derivation path, this is unnecessarily restrictive. There isn't even a guarantee that an address does not have scripthash components.

Also, consider that address indices also exist.

https://cips.cardano.org/cips/cip1852/

Reply:
No I mean:

>An arbitrary string that serves as the identifier of the wallet account connected to the DApp, formed by the wallet according to its own rules. The main point is that the wallet understands what is required of it when it receives this identifier in a request.

It's just an internal identifier. It can be anything that a specific wallet can recognize.